### ISSUE TITLE
[Anon] Bug report: Adeptus Astartes.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-killteam#362]
**File:** Adeptus Astartes.catz

**BattleScribe version:** 2.02.03

**Platform:** Android

**Dropbox:** No

**Description:** When attempting to make a list.  It times out and claims my phone doesn't have enough RAM.  If I turn on low RAM mode and it still says it ran out of RAM

**Comment by Mad-Spy** [comment]
Duplicate of #321. We're waiting on a fix from the developer.
